.github/workflows/ci.yaml

```diff
@@ -19,30 +19,8 @@ jobs:
         with:
           go-version-file: "go.mod"
 
-      # Python
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version-file: ".python-version"
-
-      - name: Install Black
-        run: pip install black
-
-      # Ruby
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@v1
-
-      - name: Install Rubocop
-        run: gem install rubocop
-
-      # Javascript
-      - name: Set up Javascript
-        uses: actions/setup-node@v4
-        with:
-          node-version-file: ".node-version"
-
-      - name: Install Prettier
-        run: npm install prettier
```

Comment:
Don't remove those

Reply:
I was waiting to merge from master again, once PR #17 was complete. Will do of course.